keymaps/ava.cson

```diff
@@ -0,0 +1,3 @@
+'atom-workspace':
+  'ctrl-alt-a': 'ava:toggle'
+  'ctrl-alt-x': 'ava:run'
```

Comment:
Is there really any point in having two separate shortcuts? I would think it's enough with just Ctrl+Alt+A which toggles and runs tests.